extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcTenantConfig.java

```diff
@@ -119,6 +125,40 @@
 
     }
 
+    @ConfigGroup
+    public static class Logout {
+
+        /**
+         * The logout endpoint at the application. If provided, the application is able to initiate the logout through this
```

Comment:
Please replace `The logout endpoint at the application` with something like `The relative path of the logout endpoint...`